translator/translate/otel/receiver/prometheus/translator.go

```diff
@@ -0,0 +1,102 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package prometheus
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/aws/amazon-cloudwatch-agent/translator/util"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/confmap"
+	"go.opentelemetry.io/collector/receiver"
+	"gopkg.in/yaml.v3"
+
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/common"
+)
+
+const (
+	otelConfigParsingError = "has invalid keys: global"
+	defaultTlsCaPath       = "/etc/amazon-cloudwatch-observability-agent-cert/tls-ca.crt"
+)
+
+var (
+	configPathKey = common.ConfigKey(common.MetricsKey, common.MetricsCollectedKey, common.PrometheusKey, common.PrometheusConfigPathKey)
+)
+
+type translator struct {
+	name    string
+	factory receiver.Factory
+}
+
+type Option func(any)
+
+var _ common.Translator[component.Config] = (*translator)(nil)
+
+func NewTranslator(opts ...Option) common.Translator[component.Config] {
```

Comment:
It doesn't look like we use the `name` or the `opts`. We can just omit them until we need them.